### ISSUE TITLE
L2 Norm

### ISSUE THREAD

**Issue by jiangtongli** [allenai/bilm-tf#60]
Sorry to borrow again!
In the paper, you said that the L2 norm is add while training the model, but I didn't find the code in the training code(training.py). Would you like to tell me where the L2 norm is added in your training code?

**Comment by matt-peters** [comment]
In some cases, we add L2 norm to the layer weights when using in the task specific model.  You can see this here: https://github.com/allenai/bilm-tf/blob/master/bilm/elmo.py#L13

To use this in your training code, you will need to add this term to your loss.
